ddtrace/tracer/tracer.go

```diff
@@ -622,3 +624,10 @@ func startExecutionTracerTask(ctx gocontext.Context, span *span) (gocontext.Cont
 	rt.Log(ctx, "span id", strconv.FormatUint(span.SpanID, 10))
 	return ctx, task.End
 }
+
+func (t *tracer) hostname() string {
+	if t.config.disableHostnameDetection {
+		return hostname.Get()
+	}
+	return ""
```

Comment:
```suggestion
	if !t.config.disableHostnameDetection {
		return hostname.Get()
	}
	return ""
```

It's often easier and less error prone to have this kind of booleans describe enabling the feature. Any reason why we'd prefer `disableHostnameDetection` instead of `enableHostnameDetection` here ?
Even for consistency, all the existing booleans in the `config` struct describe enabling features/options.